test/units/index.js

```diff
@@ -7,4 +7,5 @@ module.exports = [
 	require('./fetchroles'),
 	require('./roomcapacity'),
 	require('./thiefcard'),
+	require('./mongoTest')
```

Comment:
CI failed here. The source file doesn't exist.

Reply:
The functionality for recording user info is not ready, so I missed checking this file.
Correct with new commit.